pkg/logs/processor/processor_test.go

```diff
@@ -113,6 +113,15 @@ func TestMask(t *testing.T) {
 	shouldProcess, redactedMessage = p.applyRedactingRules(newMessage([]byte("The credit card 4323124312341234 was used to buy some time"), &source, ""))
 	assert.Equal(t, true, shouldProcess)
 	assert.Equal(t, []byte("The credit card [masked_credit_card] was used to buy some time"), redactedMessage)
+
+	source = newSource("mask_sequences", "${1}[masked_value]", "([Dd]ata_?values=)\\S+")
```

Comment:
can we please add test when there is nothing after the `=`please:
`data_values=` should return `data_values=` and not `data_values=[masked_value]`, right ?

Reply:
Added. :) Indeed, for that test the regex fails to match so it doesn't perform the replacement.

I posted this in the logs T2 channel so @NBParis has seen it. 👍